Neos.ContentRepository.Core/Classes/DimensionSpace/DimensionSpacePoint.php

```diff
@@ -61,10 +61,19 @@ public static function fromJsonString(string $jsonString): self
         return self::instance(json_decode($jsonString, true));
     }
 
+    /**
+     * Creates a dimension space point for a simple zero-dimensional content repository.
+     */
+    public static function zeroDimensional(): self
```

Comment:
Thank you for this, this is so much more explicit than the `fromArray([])` calls.
I just wonder about the naming.
In other places we use `create...` (for example `createEmpty()`) to emphasize that this is a factory method..

OTOH and "empty" DSP is maybe not much clearer..

Reply:
Suggestion: `createWithoutDimensions`